assets/js/main.js

```diff
@@ -1092,6 +1087,12 @@ function update() {
   setTimeout(update, 30000);
 }
 
+let resizeId;
+$('.resize-trigger').click(() => {
```

Comment:
Binds to 0 elements, `.resize-trigger` doesn't exist yet. [This](https://stackoverflow.com/a/33092954/2428152) might help.

Reply:
_fek_ i just need to add it on ready instead of immediate.

Reply:
yeah, that should help as well